src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -245,6 +256,8 @@ def _snapshot(self, fetcher, loader) -> list[any]:
 
     def _append_records(self, items):
         if len(items) == 0:
+            if self._klass is not None:
```

Comment:
These two lines could be moved to the top of the method 🤷🏻‍♂️

Reply:
I'm not sure what you mean.

Reply:
Fixed